liblouis/maketable.c

```diff
@@ -90,18 +90,28 @@ printRule(TranslationTableRule *rule, widechar *rule_string) {
 		return 0;
 	default: {
 		int l = 0;
+		if (rule->nocross) {
```

Comment:
Can't you do some sort of memcpy or strcopy here, instead of copying the chars individually?

Reply:
Sure. That's what I was doing at some point but ended up doing it like this (for some other reason). I think it doesn't look so bad so left it.